### DISCUSSION TITLE
Connect/Link two "Target" elements together??

### DISCUSSION THREAD

**Discussion by PianoGeek15** [helgoboss/helgobox#1374] [Ideas]
It would be AWESOME to have the ability to link two Reaper elements together, and be able to control either one from the other one (a two-way link). Basically, instead of a usual ReaLearn mapping which has a source and a target, it would be two targets linked together somehow (Ideally you would have the same "Glue" options, but this might be complicated to set up). 

I see a lot of use cases for this feature. Whether you use it to automatically mute a track when its volume is turned to 0, or you use the "Relative" option to turn a bunch of tracks up or down by the same amount, this would be a great feature to have!

**Comment by helgoboss** [comment]
If this will ever be supported, then via #634 (the ability to use targets as source). In order to not clutter the idea section, I will close this as duplicate. Feel free to discuss in #634.

BTW, ReaLearn already supports a subset of this: Macro parameters (controlling many targets at once with just one ReaLearn compartment parameter). Compartment parameters are one of the few targets which can also be used as source.

Personally, I like the macro parameter approach much more than a random spaghetti of bi-directional target links. It's clear what triggers what. And with this approach, it would be even possible to implement real-time target invocation and rendering support, in the future. That's sort of impossible when linking arbitrary targets.
